### PR TITLE
Add readme in system module

### PR DESCRIPTION
**Pull request by Bhashinee**
## Purpose
$subject.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+
+Execute the commands below to build from source.
+
+1. To build the library:
+
+        ./gradlew clean build
+
+2. To run the integration tests:
+
+        ./gradlew clean test
```
Comment (chamil321) [inline README.md:36]: Did you test this command? I think we need to add `test` task to build gradle to make this work

Comment (Bhashinee) [reply README.md:36]: Added the task to the gradle

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
-# module-ballerina-system
\ No newline at end of file
+Ballerina System Library
+===================
+
+  [![Build](https://github.com/ballerina-platform/module-ballerina-system/workflows/Build%20master%20branch/badge.svg)](https://github.com/ballerina-platform/module-ballerina-system/actions?query=workflow%3ABuild)
```
Comment (niveathika) [inline README.md:4]: Shall we rename this GitHub action as just build? Else the badge will be really long

https://github.com/ballerina-platform/module-ballerina-system/blob/stdlib-configs/.github/workflows/build-master.yml#L1

Comment (Bhashinee) [reply README.md:4]: Renamed as suggested

**Review by niveathika — APPROVED** [review]
LGTM

File: system-ballerina/build.gradle [hunk 3]
```diff
@@ -128,7 +128,38 @@ task revertTomlFile {
...
+        }
+    }
+}
+
+test {
+    dependsOn(ballerinaTest)
+}
+
 task ballerinaBuild {
+    dependsOn(test)
```
Comment (niveathika) [inline system-ballerina/build.gradle:162]:
```suggestion
    dependsOn(copyStdlibs)
    dependsOn(":system-native:build")
    dependsOn(test)
```
Add these dependency too. Else if you skip these will also not run.

File: system-ballerina/build.gradle [hunk 4]
```diff
@@ -137,23 +167,19 @@ task ballerinaBuild {
...
         def additionalBuildParams = ""
 
-        if (project.hasProperty("skipBallerinaTests")) {
-            additionalBuildParams += "--skip-tests"
-        }
-
-        if (project.hasProperty("debugBallerina")) {
-            def port = project.getProperty("debugBallerina").toString();
-            additionalBuildParams += "--debug ${port}"
+        if (project.hasProperty("debug")) {
```
Comment (MadhukaHarith92) [inline system-ballerina/build.gradle:170]: Unnecessary. @niveathika WDYT.

Comment (Bhashinee) [reply system-ballerina/build.gradle]: Yes, this is not required. Removed.

**Review by MadhukaHarith92 — APPROVED** [review]

**Pull request merged**